WORKSPACE

```diff
@@ -33,26 +33,44 @@ bind(
     actual = "@submodule_gtest//:gtest",
 )
 
+bind(
+    name = "gflags",
+    actual = "@com_github_gflags_gflags//:gflags",
+)
+
 new_local_repository(
     name = "submodule_boringssl",
-    path = "third_party/boringssl-with-bazel",
     build_file = "third_party/boringssl-with-bazel/BUILD",
+    path = "third_party/boringssl-with-bazel",
```

Comment:
Ok, makes sense. I thought this might have something to do with my inability to build. I was able to munge around with the submodules some more and get sanity to pass but the build still fails, so I guess this is unrelated.